src/@types/navigation.d.ts

```diff
@@ -0,0 +1,68 @@
+import {DrawerScreenProps as DefaultDrawerScreenProps} from '@react-navigation/drawer';
+import {
+  CompositeScreenProps,
+  NavigatorScreenParams,
+} from '@react-navigation/native';
+import {NativeStackScreenProps} from '@react-navigation/native-stack';
+
+declare global {
+  namespace ReactNavigation {
+    interface RootParamList extends RootNativeStackParamList {}
```

Comment:
can this namespace be stored in one place? RootDrawerParamList is also being extended to RootParamList in global.d.ts